test/test_autograd.py

```diff
@@ -3482,6 +3482,13 @@ def test():
         test()
         self.assertEqual(dealloc[0], 1)
 
+    def test_inplace_view_leaf_errors(self):
+        # Issue #21875: Fail faster (when we try to modify the view vs. in backward())
+        x = torch.zeros(1, requires_grad=True)
+        y = x
+        with self.assertRaises(RuntimeError):
```

Comment:
You can use `assertRaisesRegex()` here with the content of the error message. That way you can be sure that the right error is raised.
And I would expect this test to fail without the view fix above.
```suggestion
        with self.assertRaisesRegex(RuntimeError, "a view of a leaf Variable that requires grad"):
```